### PR TITLE
release: design fixes

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
synced with Michelle

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx [hunk 1]
```diff
@@ -91,8 +91,11 @@ function HistoryOpScreenInner({
 
   return (
     <View style={ss.container.screen}>
-      <ScreenHeader title="Transfer" onExit={leaveScreen} hideOfflineHeader />
-      <Spacer h={16} />
+      <ScreenHeader
+        title={getOpVerb(op, account.address)}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/HistoryOpScreen.tsx:95]: good call!

File: apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx [hunk 2]
```diff
@@ -69,9 +69,10 @@ export const SwipeUpDown = forwardRef<SwipeUpDownRef, SwipeUpDownProps>(
 
     const snapPoints = useMemo(() => {
       if (snapPointCount === 2) {
-        return [500, maxHeightOffset - tabBarHeight];
+        // TODO: use dynamic sizing on TransferOp screen
+        return [440, maxHeightOffset - tabBarHeight];
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx:73]: If you make a payment link, then navigate to cancel it, is the cancel button still visible?

File: apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx [hunk 3]
```diff
@@ -74,16 +75,19 @@ export function OwnRequestBottomSheet({
...
+        onExit={() => {
+          dispatcher.dispatch({ name: "hideBottomSheet" });
+        }}
+        hideOfflineHeader
+      />
+      <TitleAmount amount={dollarsToAmount(reqStatus.link.dollars)} />
+      <Spacer h={8} />
       <TextCenter>
-        <TextH3>Requested</TextH3>
+        <TextBodyCaps color={color.grayMid}>USDC • BASE</TextBodyCaps>
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx:88]: wouldn't hardcode it. see SettingsScreen.tsx AccountHeader

File: apps/daimo-mobile/src/view/shared/HistoryList.tsx [hunk 4]
```diff
@@ -91,8 +91,7 @@ export function HistoryListSwipe({
 
   // Easy case: show a fixed, small preview list
   if (maxToShow != null) {
-    const title =
-      otherAcc == null ? "Transaction history" : `Transactions between you`;
+    const title = otherAcc == null ? "Recent activity" : `Between you`;
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/shared/HistoryList.tsx:94]: much better

File: apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx [hunk 5]
```diff
@@ -74,16 +75,19 @@ export function OwnRequestBottomSheet({
 
   return (
     <View style={ss.container.padH16}>
-      <Spacer h={16} />
+      <ScreenHeader
+        title="You requested"
+        onExit={() => {
+          dispatcher.dispatch({ name: "hideBottomSheet" });
+        }}
+        hideOfflineHeader
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx:83]: 👍

**Review by dcposch — APPROVED** [review]
LGTM after comments

File: apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx (hunk 2 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx:73]: yes, checked
Simulator Screenshot - iPhone 14 Pro - 2024-04-03 at 14 41 17

File: apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx (hunk 3 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/sheet/OwnRequestBottomSheet.tsx:88]: fixed

**Pull request merged**